en/docs/reference/synapse-properties/expressions.md

```diff
@@ -1,376 +1,109 @@
 
-In WSO2 Micro Integrator, expressions are a way to dynamically compute values within various integration artifacts such as sequences, proxies, endpoints, and APIs. These expressions are used to extract, transform, and route data as it flows through the integration logic.
+In WSO2 Micro Integrator, expressions are a way to dynamically compute values within various integration artifacts such as sequences, APIs, endpoints, and proxies. These expressions are used to extract data as it flows through the integration logic.
 
-## XPath expressions
-
-The WSO2 Micro Integrator supports standard XPath functions and variables through its underlying XPath engine. It supports XPath 1.0 by default where as the support for XPath 2.0 can be introduced by adding the following property in `<MI_HOME>/conf/deployment.toml`.
-
-```toml
-[mediation]
-synapse.enable_xpath_dom_failover=true
-```
-
-The Micro Integrator also provides custom XPath functions and variables for accessing message properties.
-
-### XPath extension functions
-
-In addition to standard XPath functions, the Micro Integrator supports the following custom functions for
-working with XPath expressions:
-
-#### `base64Encode()` function
-
-The `base64Encode` function returns the base64-encoded value of the
-specified string.
-
-Syntax:
-
--   `           base64Encode(string value)          `
--   `           base64Encode(string value, string charset)          `
-    `                     `
-
-#### `base64Decode()` function
-
-The `base64Decode` function returns the original value of the specified
-base64-encoded value.
-
-Syntax:
-
--   `           base64Decode(string encodedValue)          `
--   `           base64Decode(string           encodedValue           , string charset)          `
-    `                     `
+The most commonly used expression languages in WSO2 MI are [Synapse Variables](#synapse-variables), [JSONPath (for JSON)](#jsonpath-expressions), and [XPath (for XML)](#xpath-expressions).
 
-#### get-property() function
+## Synapse variables
 
-The `get-property()` function allows any XPath expression used in a configuration to look up information from the current message context. Using the [Property mediator]({{base_path}}/reference/mediators/property-mediator), you can retrieve properties from the message context and header.
+There is a set of predefined synapse variables that you can use
+to extract data from the message. These synapse variables in different [scopes]({{base_path}}/reference/synapse-properties/scopes) are as follows:
 
-The syntax of the function takes the following format.
+### $ctx
 
--   `          get-property(String propertyName)         `
--   `          get-property(String scope, String propertyName)         `
+The prefix used to get properties set at the [default scope]({{base_path}}/reference/synapse-properties/scopes/#default-scope). For example, to get the value of property with name `ERROR_MESSAGE`, use the expression `$ctx:ERROR_MESSAGE`.
```

Comment:
Shall we reword this as one of the following?

- For example, to get the value of the property with the `ERROR_MESSAGE` name, use the `$ctx:ERROR_MESSAGE` expression.

or

- For example, to get the value of the `ERROR_MESSAGE` property name, use the `$ctx:ERROR_MESSAGE` expression.

Reply:
No it won't express the fact that we highlight here.